lib/mirage/mirage_impl_random.ml

```diff
@@ -1,40 +1,25 @@
 open Functoria
+open Mirage_impl_misc
```

Comment:
Why do you need `Mirage_impl_misc` here?

Reply:
it was only copy + paste from the other branch ;) and since `dune build` didn't complain, i suspect it is needed for something (but maybe it is not)

Reply:
got rid of the global open